go/app/main.go

```diff
@@ -89,20 +89,48 @@ func addItem(c echo.Context) error {
 		return err
 	}
 
-	// add item to list
-	itemlist.Items = append(itemlist.Items, item)
+	//データベースを開く
+	db, err := sql.Open("sqlite3", "/Users/hinako/mercari-build-training/db/mercari.sqlite3")
```

Comment:
[nits] 
L21くらいにconstで定義して変数でおくのもいいかも
```
const DB_PATH = "/Users/hinako/mercari-build-training/db/mercari.sqlite3"
```

```suggestion
	db, err := sql.Open("sqlite3", DB_PATH)
```

Reply:
[ab95dbb](https://github.com/hinako1217/mercari-build-training/pull/3/commits/ab95dbb754138df0d24844723407d6f05f51391b)
変更しました